Algebraic_foundations/include/CGAL/number_utils.h

```diff
@@ -324,6 +324,36 @@ NT approximate_sqrt(const NT& nt)
   return approximate_sqrt(nt, Sqrt());
 }
 
+template <class NT>
+Comparison_result
+compare(const NT& xnum, const NT& xden,
+        const NT& ynum, const NT& yden)
```

Comment:
I think you must rename that function template. Its behavior is not obvious from the name and the signature, and adding an overload to the overload set of `compare` will only degrade compilation performances, without any benefit. Maybe `compare_quotients`?

Reply:
done.